crates/ere-sp1/src/lib.rs

```diff
@@ -13,6 +13,8 @@ use zkvm_interface::{
     ProverResourceType, zkVM, zkVMError,
 };
 
+include!(concat!(env!("OUT_DIR"), "/sdk_version.rs"));
```

Comment:
This looks okay, in the future we could make it such that each zkVM needs to implement this by adding methods to the interface

Reply:
Ah, adding `name()` and `sdk_version()` to some trait was my first option, but I wasn't sure if an existing one would be fine.

Do you think adding those methods to `zkVM` might be ok?

Reply:
@kevaundray, another option is the `Compile` trait, but feels like from the existing traits `zkVM` (which contains `execute`, `prove`, `verify`) might be the best.